Mathlib/FieldTheory/PurelyInseparable/Tower.lean

```diff
@@ -3,19 +3,24 @@ Copyright (c) 2024 Jz Pan. All rights reserved.
 Released under Apache 2.0 license as described in the file LICENSE.
 Authors: Jz Pan
 -/
+import Mathlib.FieldTheory.LinearDisjoint
 import Mathlib.FieldTheory.PurelyInseparable.PerfectClosure
 
 /-!
 
 # Tower law for purely inseparable extensions
 
-This file contains results related to `Field.finIsepDegree` and the tower law.
+This file contains results related to `Field.sepDegree`, `Field.insepDegree` and the tower law.
 
 ## Main results
 
 - `Field.lift_sepDegree_mul_lift_sepDegree_of_isAlgebraic`: the separable degrees satisfy the
   tower law: $[E:F]_s [K:E]_s = [K:F]_s$.
 
+- `Field.lift_insepDegree_mul_lift_insepDegree_of_isAlgebraic`:
+  `Field.finInsepDegree_mul_finInsepDegree_of_isAlgebraic`: the inseparable degrees satisfy the
+  tower law: $[E:F]_i [K:E]_i = [K:F]_i$.
```

Comment:
Might be better to switch to `ₛ` and `ᵢ` throughout the file?

Reply:
This is in `$` so it's rendered as LaTeX. If use `ₛ` and `ᵢ` then I should use ` instead. These characters look too small IMHO.